modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,61 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run commands in parallel on a large number of minions.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting parameters to control the load in your environment.
+
+These parameters are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+[NOTE]
+====
+Salt commands that are executed from the command line are not subject to these parameters.
+====
+
+
+
+== Batching
+
+There are two parameters that control how actions are sent to clients, one for the batch size, and one for the delay.
+
+When the Salt master sends a batch of actions to the target minions, it will send it to the number of minions determined in the batch size parameter.
```

Comment:
Hmm, technically all the Salt master sends are commands or function calls. Actions are a `{productname}`-specific concept. Given that batching is a `{productname}`-specific concept too, I would reword that as:
```suggestion
When the {productname} Server executes an action against multiple target minions, it will send a Salt command to the number of minions determined in the batch size parameter.
```